.github/workflows/terraform-observe_prerelease.yaml

```diff
@@ -6,7 +6,7 @@ on:
         required: true
       TERRAFORM_MODULES_REGION:
         required: true
-      TERRAFORM_MODULES_SLACK_URL:
+      TERRAFORM_MODULES_PRERELEASE_SLACK_URL:
```

Comment:
I created separate secrets for the release and prerelease webhooks so that we can modify the message that gets posted and the channel it gets posted to depending on whether the notification is for an official release or just a prerelease build.